cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -88,9 +97,19 @@ class alignas(16) column_device_view_base {
  public:
   column_device_view_base()                               = delete;
   ~column_device_view_base()                              = default;
-  column_device_view_base(column_device_view_base const&) = default;
-  column_device_view_base(column_device_view_base&&)      = default;
+  column_device_view_base(column_device_view_base const&) = default;  ///< Copy constructor
+  column_device_view_base(column_device_view_base&&)      = default;  ///< Move constructor
+  /**
+   * @brief Copy assignment operator
+   *
+   * @return Reference to this object
+   */
   column_device_view_base& operator=(column_device_view_base const&) = default;
+  /**
+   * @brief Move assignment operator
+   *
+   * @return Reference to this object (after transferring ownership)
```

Comment:
Do we really need to manually document all move and copy assignment operators? Is there any way to ask doxygen to just ignore them, or autogenerate them? there are very few cases where I think that they need specialized documentation (e.g. to indicate exactly what resources are moved).

Reply:
doxygen does not recognize `= default` constructors and assignment operators yet. The workarounds like doxygen aliases, INPUT_FILTER are not really preferable.

Reply:
Got it. Is there work under development to improve this?